package.json

```diff
@@ -192,12 +192,12 @@
             "type": "string"
           },
           "default": [],
-          "description": "Optional array of strings to pass to the language server. Only works with Dafny versions above 3.10. Available options can be seen using 'dafny server --help'"
+          "description": "Optional array of strings to pass to the language server. Only works if the Dafny version is 3.10 or greater. Available options can be seen using 'dafny server --help'"
         },
         "dafny.cliPath": {
           "type": "string",
           "default": null,
-          "description": "Path to the Dafny CLI. Only used when 'custom' is the chosen Dafny version. The Dafny version must be 3.10 or greater."
+          "description": "Path to Dafny.exe, Dafny.dll or DafnyLanguageServer.dll. Only used when 'custom' is the chosen Dafny version."
```

Comment:
I don't think it works with `DafnyLanguageServer.dll` or versions older than 3.10, since for custom it always uses the new type of CLI arguments. It might sort of work but without using all the preferences.

Reply:
Could you please at least test it and ensure it works? Without being able to use DafnyLanguageServer.dll it would be difficult, for example, to bissect the language server to detect a regression issue. There might be workarounds by using nightly releases but I wouldn't rely on them as they might be overriden.

Reply:
I tested it and it works but indeed only with the default options, so the preferences are ignored.